app/assets/javascripts/compute_resource.js

```diff
@@ -249,12 +249,13 @@ function vsphereGetResourcePools(item) {
     data: data,
     complete: function() { $(item).indicator_hide();},
     success: function(request) {
-      $('#compute_attribute_vm_attrs_resource_pool').select2('destroy').empty();
+      selectbox = $('*[id*=resource_pool]')
+      selectbox.select2('destroy').empty();
```

Comment:
Maybe remove the destory.empty function a little bit up (to around https://github.com/timogoebel/foreman/blob/fix_resource_pool_js/app/assets/javascripts/compute_resource.js#L245) so when changing, the select will be empty until populated again with the new data?

Other than that, works great.